src/main/java/seedu/address/model/person/MatriculationYear.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.Year;
+
+/**
+ * Represents a Person's matriculation year in the address book.
+ * Guarantees: immutable; is always valid
```

Comment:
Perhaps you could elaborate how valid is defined?

Reply:
Thanks for the suggestion! I have updated the code accordingly with the suggested changes.